gulpfile.js

```diff
@@ -493,6 +521,20 @@ function makeRef(done, bot) {
   });
 }
 
+gulp.task("sandbox", function (done) {
```

Comment:
I'd suggest separating the `createScriptingBundle` from this gulp-task, and possible even remove this completely.

For the `mozcentral` task you added the `createScriptingBundle` to list existing list of tasks in https://github.com/mozilla/pdf.js/blob/55f55f58594b9a6947fecaabf8ef4e3b02002023/gulpfile.js#L1065-L1067
hence I suggest adding `createScriptingBundle` *and* `createSandboxBundle` to `buildGeneric` instead: https://github.com/mozilla/pdf.js/blob/55f55f58594b9a6947fecaabf8ef4e3b02002023/gulpfile.js#L723-L729

Reply:
It's on purpose.
The idea is to get a bundle from `initialization.js` and the convert the bundle content into a string and inject this string in the sandbox bundle. This string will be eval-ed in the sandbox in order to create all what we need for the JS api.
But of course, if you see a better way to do, please tell me.